slides/05_collaboration.qmd

```diff
@@ -319,23 +319,23 @@ gitGraph
 sequenceDiagram
```

Comment:
in the presentation html the text could not be read clearly
![image](https://user-images.githubusercontent.com/36079400/226792183-436063e0-7ebc-4f07-9cbe-ae9e6b078f4d.png)

Reply:
is there box size option? maybe we can also make a first pass now and see how it renders on the github pages site, locally can look different

Reply:
It looks okay in my end
![image](https://user-images.githubusercontent.com/48791914/226793300-1ca6fe0a-c9e2-4a7c-bc2d-495d6436dc98.png)

Reply:
then no worries